src/widgets.js

```diff
@@ -179,9 +203,5 @@ function renderUrl(url, props, elem, overrides, force) {
 }
 
 export {
-  define,
-  isDefined,
-  load,
-  render,
-  renderUrl,
+  define, isDefined, load, render, renderUrl,
```

Comment:
Dit mag op één lijn blijven

Reply:
Dan klaagt eslint: Expected a line break before this closing brace.eslint(object-curly-newline)